test/games/prime1/exporter/test_prime_game_exporter.py

```diff
@@ -67,6 +76,11 @@ def test_patch_game(mocker, tmp_path, use_echoes_models):
     else:
         assert not asset_cache_path.exists()
 
+    if use_enemy_attribute_randomizer:
+        mock_enemy_data.assert_called_once_with("inputIso", "outputIso", "seed", 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, False)
```

Comment:
`tmp_path.joinpath("input.iso")` instead of `"inputIso"`
`tmp_path.joinpath("output.iso")` instead of `"outputIso"`
whatever you pass as a seed instead of `"seed"`